### ISSUE TITLE
[Feature idea] export Highlighted pieces

### ISSUE THREAD

**Issue by rotello** [chetachiezikeuzor/Highlightr-Plugin#70]
Hello, 
I ve just discovered your plugin, which is super slick and so far i Like it.
I ve been using Obsidian for a year or so now and I am slowly trying to use Tiago Forte System. I like the "distill" idea, and the highlight part is an important piece.

I wonder if possible to add a "export highlighted pieces" function. it could be a simply "copy / past only the highighted part" or a export into new note , or export into the same note.... but i think it would be great for progressive summarization.

ciao, and thanks!

**Comment by AMGMNPLK** [comment]
There is a plugin called Extract Highlights that do something similar, but it is bit limited. I would love to have this integrated both together and with callouts

**Comment by magrubertone** [comment]
Yes this would be nice! Thanks for the plugin by the way! I will investigate Extract Highlights.
